Intersect.Client/Maps/MapInstance.cs

```diff
@@ -1260,7 +1260,7 @@ public void DrawActionMsgs()
                     ActionMessages[n].Y * Options.TileHeight -
                     Options.TileHeight *
                     2 *
-                    (1000 - (ActionMessages[n].TransmissionTimer - Timing.Global.Milliseconds)) /
+                    (1000 - (ActionMessages[n].TransmissionTimer - Timing.Global.MillisecondsUtc)) /
```

Comment:
I am not sure about this being not-synchronized

Reply:
this timer is used to **_graphically_ render the strings of action messages** on screen going up and fading out, it doesn't matter if the rendered position of these aren't synced in time, just like animations, it should be good as long as the user can see the action messages on their side.

Preview (using MillisecondsUtc):

https://user-images.githubusercontent.com/17498701/234128115-d59dfff7-29da-456c-a854-1968f46c8c84.mp4